packages/toc/src/generators/notebook/twist_button.tsx

```diff
@@ -21,7 +21,11 @@ type onClick = (cellRef?: Cell) => void;
  * @param onClick - "click" handler
  * @returns rendered twist button
  */
-function twistButton(cellRef: Cell, collapsed: boolean, onClick: onClick) {
+function twistButton(
```

Comment:
Should this be a typed object as a parameter?

Reply:
So that'd be something like `ITwistButtonProps` that has fields `heading`, `collapsed`, and `onClick`? Or do you think I should just have it take two arguments: one being `ICollapseChangedArgs` and the other being `onClick`, even though it wouldn't be accessing the `tocType` field of the `ICollapseChangedArgs`?

Reply:
I like `ITwistButtonProps`